### PR TITLE
chapter8: login with session, using form_for to render form,...

### PR DESCRIPTION
**Pull request by giapbnv-0886**

### CODE REVIEW

**Comment by giapbnv-0886** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user && user.authenticate(params[:session][:password])
```
Comment (huuphuc93) [inline app/controllers/sessions_controller.rb]:
```suggestion
    if user&.authenticate(params[:session][:password])
```

**Comment by giapbnv-0886** [comment]
ready

**Comment by huuphuc93** [comment]
merged

**Pull request merged**